dirs/dirs.go

```diff
@@ -190,6 +198,39 @@ func init() {
 	SetRootDir(root)
 }
 
+// Returns a slice of the home directories currently in use.
+func GetSnapHomeDirs() []string {
```

Comment:
this shouldn't have a "Get" in the name, also the doc comment should start with the function name